src/jit/FloatMathInl.h

```diff
@@ -498,6 +498,14 @@ static bool emitFloatCompare(sljit_compiler* compiler, Instruction* instr)
         if (nextInstr->opcode() != ByteCode::SelectOpcode) {
             ASSERT(nextInstr->opcode() == ByteCode::JumpIfTrueOpcode || nextInstr->opcode() == ByteCode::JumpIfFalseOpcode);
 
+            sljit_emit_fop1(compiler, opcode, params[0].arg, params[0].argw,
```

Comment:
This is always executed. We should not come here when flag needs to be stored, so `*operand == VARIABLE_SET_PTR(nullptr)` must be true.